jsx/Form.js

```diff
@@ -1552,7 +1552,7 @@ class ButtonElement extends Component {
 
   render() {
     return (
-      <div className="row form-group">
+      <div className={this.props.className}>
```

Comment:
What happens if the "row" and "form-group" are missing from the div? Is that something that might sometimes be desirable, or would it completely break the rendering (in which case, maybe it should be passing in additional classes that it merges with them instead of an exhaustive list of class names)

Reply:
Hey @driusan, the defaultProps will populate the "row form-group" as default if className isn't supplied. 
```
ButtonElement.defaultProps = {
  className: 'row form-group
}
```
The change made by @zaliqarosli has the ability to then add custom css class for when we don't want to use "row form-group" and a few cases around LORIS this is useful.

Reply:
I know what the change she did it, I'm asking if that's the right change.

Reply:
@driusan I think it is the right change.

Reply:
@driusan removing `row form-group` removes right, left, and bottom margins from the button. so those classes are desirable when you want to line up the button element with the rest of the elements in <FormElement />. However, there are times when you don't want the margins because you want to fit the ButtonElement in some where that is not a row in the form. Having said that though, I no longer need this case. And realized that we have the CTA (call to action) element that doesn't have formatting. so i think i'll close this PR